tests/38-basic-pfc_coverage.c

```diff
@@ -81,6 +81,12 @@ int main(int argc, char *argv[])
 	if (rc < 0)
 		goto out;
 
+	/* bug #117 - seccomp_export_pfc hangs on prioritized syscalls
+	 * with no rules */
```

Comment:
I try not to reference GitHub specific numbers since those really only have meaning in GitHub and who knows where this code might be 5, 10, 15 years from now.

No need to respin, just something to keep in mind for future patches.